src/modules/auth/credentials/CredentialsCreateContainer.js

```diff
@@ -238,9 +239,9 @@ function CredentialsCreateContainer(props) {
 										{(activeTab == "create") && (config !== undefined) && config.creation.map((item, idx) => {
 											switch(item.type) {
 												case 'username': return(<UserNameField key={idx} register={register} config={item} getValues={getValues} errors={errors} required={item.policy === "required"} />)
-												case 'email': return(<EmailField key={idx} register={register} config={item} getValues={getValues} errors={errors} required={item.policy === "required"} />)
+												case 'email': return(<EmailField key={idx} register={register} config={item} getValues={getValues} errors={errors} required={true} />)
```

Comment:
@Samsunggora Maybe some misunderstanding, but required field should be taken from the info provided by backend servivce, so why did you hardcode it to be always true?

It should be: `required={item.policy === "required"}`